packages/owl-bot-post-processor/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": "./node_modules/gts/tsconfig-google.json",
+  "compilerOptions": {
+    "lib": ["es2018"],
+    "esModuleInterop": true,
```

Comment:
This module interop stuff is also for the benefit of probot, it's different than the config we use when generating a new client library.